postgrest/base_request_builder.py

```diff
@@ -109,6 +110,8 @@ class APIResponse(BaseModel):
     """The data returned by the query."""
     count: Optional[int] = None
     """The number of rows returned."""
+    error: Optional[APIError] = None
```

Comment:
I'd rather raise an APIError instead of returning an APIResponse with an error field, the error field feels like a JS thing 😄